ascify art/Ascify-Art.py

```diff
@@ -546,12 +576,12 @@ def changedir():
     if not dir_:
         return
     if len(dir_)>=30:
-        entry_location.configure(text=dir_[:25]+"..."+dir_[-5:])
+        entry_location.configure(text=f"{dir_[:25]}...{dir_[-5:]}")

```

Comment:
Function `changedir` refactored with the following changes:

- Use f-string instead of string concatenation [x2]